src/widgets/JSONForm/JSONForm.js

```diff
@@ -207,6 +207,16 @@ export const JSONFormComponent = React.forwardRef(
       []
     );
 
+    const mounted = useIsMounted();
+    const [noValidate, setNoValidate] = React.useState(true);
+    const touchedFields = {};
```

Comment:
using `State` caused the form to reload, losing entered values.. so went with a mutable object. Not ideal, but working.

Reply:
Maybe better to `useRef` so that there isn't future hunchy funchy with `memo`ing - but I don't think it's needed right now

Reply:
good point!